.github/workflows/publish.yml

```diff
@@ -33,6 +33,10 @@ jobs:
       uses: codecov/codecov-action@v1
     - name: Pack
       run: dotnet pack --configuration Release --verbosity normal
+    - name: Pack (DiffSharp-cuda-linux)
+      run: dotnet pack --configuration Release --verbosity normal bundles\DiffSharp-cuda-linux
```

Comment:
Needs to be `/` (same below): causing the packaging to fail: https://github.com/DiffSharp/DiffSharp/runs/1086050946?check_suite_focus=true#step:11:10

Reply:
Thanks!  fixed and CI is trying again :-)